README.md

```diff
@@ -1 +1,61 @@
-# AnimeX
\ No newline at end of file
+
+<br />
+<div >
+    <img src ="https://i.ibb.co/4gCLhXM/chick-1.png" style ="height: 150px; width: px;">
+
+<div >
+    <h2> AnimeX
+    </h2>
+</div>
+
+</div>
+
+
+## **Live Website** <img src="https://i.imgur.com/QsX6030.png" style="height: 50px; width: 50px; margin-bottom: 20px;">
+ <span id="live"></span>
+You can check the website here [**AnimeX**](https://animex-gsg.herokuapp.com/).
+
+
+## **About Our Website** <span id="about"></span>
+
+This is an anime website that features animes based on your search. Also, it provides you with GIFs that are related to the anime you like. 
+
+While the client-side programming and design of our website are top-notch, the server-side programming is on another level. By using express, we have managed to cut down our code and also improve our fetching process by using promises. 
+
+We have also managed to hide our API key through .env npm package.
+
+This made our project stand out in the backend programming it has.
+
+
+## **User Journey** :books: <span id="stories"></span>
+
+The user enters the website to find a landing page with an anime theme. Then, as the user scrolls down, he can search for an anime he likes. 
+
+Afterwards, an anime list will show up with a 'View GIFs' button that appears when hovering on the specific anime card. When clicking, the user gets redirect to our GIFs page that shows anime pictures related to the anime.
+
+# **Built with** 
+
+
+- HTML5
+- CSS3
+- JavaScript
+- APIs
+- Node.js
+- Express.js
+- Github
+- Heroku
+
+# Team Members
+- [Shatha k. Eqdaih](https://github.com/shathakh) 
+- [Abdallah Alshawaf](https://github.com/abdallah-alshawaf)
+
+# Website Layout
+
+![screen 1](https://user-images.githubusercontent.com/77805478/185121325-70cb5f27-fcd9-4134-924b-2c9bef2933c5.jpg)
+
+![screen2](https://user-images.githubusercontent.com/77805478/185121467-f170cc23-22ec-418a-b88a-63311fc8a348.jpg)
+
+![screen3](https://user-images.githubusercontent.com/77805478/185121597-1877a13a-4c39-4883-9f08-33803ab082c4.jpg)
+
+
+
```

Comment:
nice readme, but it is missing an important section, the installation guide, this section should help in how to set up your project locally and get it up and running